werk24/models/property/striae.py

```diff
@@ -63,6 +63,11 @@ class W24PropertyStriaeIso12123(W24PropertyStriae):
     grade: str = Field(examples=["SW60", "SW10"])
 
 
+class W24PropertyStriaeFreeText(W24PropertyStriae):
+    """Striae Free Text"""
+
+    free_text: str
+
```

Comment:
nee to have variation_type: Literal["FREETEXT"] = "FREETEXT"
Otherwise the deserializer doesnt know what to do

Reply:
updated